src/sgx/faasm_sgx_enclave.cpp

```diff
@@ -151,7 +168,66 @@ extern "C"
         free(msg_ptr);
         return returnValue;
     }
-#endif
+
+    faasm_sgx_status_t send_recv_msg(uint32_t thread_id, const void *send_payload_ptr, const uint32_t send_payload_len, void** recv_payload_ptr, uint32_t* recv_payload_len) {
+        sgx_wamr_msg_t* msg_ptr;
+        sgx_status_t sgx_ret_val;
+        faasm_sgx_status_t ret_val;
+        faasm_sgx_nonce_t p_iv;
+        sgx_wamr_msg_t* response_ptr;
+        uint8_t *decrypted_payload;
+        sgx_wamr_msg_nonce_payload *nonce_payload;
+        uint8_t *result;
+        if(thread_id >= _faasm_sgx_tcs_len)
```

Comment:
This should get picked up with `run_clang_format` but we don't use no-bracket `if` statements in this project, and even this file is inconsistent in its choices.